### DISCUSSION TITLE
Error messages enhancement

### DISCUSSION THREAD

**Discussion by T-prog3** [TumblThreeApp/TumblThree#274] [Ideas]
I found myself having 30 errors relating to `Error *: The blog already exist: *` and it was really tedious to click X on every single one. 

- So i thought couldn't we have a `Close all errors function` either as a new button or something like `Shift + Click X`. 
- I would also request a specification in the message relating to the collection where the already existing user is located. Something like `Error *: The blog already exist in myCollection: *` this would give me a clearer answer on _what_ and _where_ without having to manually look it up.

**Comment by thomas694** [comment]
Thanks for your feature suggestion. We'll think about it.
